cnd/src/http_api/orderbook.rs

```diff
@@ -199,14 +191,22 @@ pub async fn get_orders(facade: Facade) -> Result<impl Reply, Rejection> {
             title: None,
         };
 
+        let ethereum_amount = siren::Field {
+            name: "bitcoin_amount".to_string(),
+            class: vec!["ethereum".to_string(), "address".to_string()],
+            _type: None,
+            value: None,
+            title: None,
+        };
+
```

Comment:
So the variable is called `ethereum_amount`, the field's name is `bitcoin_amount` and the classes are `ethereum` and `address`? :D What is going on here?